app/controllers/api/v2/subnets_controller.rb

```diff
@@ -40,7 +40,7 @@ def show
           param :gateway, String, :desc => N_("Subnet gateway")
           param :dns_primary, String, :desc => N_("Primary DNS for this subnet")
           param :dns_secondary, String, :desc => N_("Secondary DNS for this subnet")
-          param :ipam, String, :desc => N_('IP Address auto suggestion mode for this subnet, valid values are "DHCP", "Internal DB", "None"')
+          param :ipam, IPAM::MODES.values, :desc => N_('IP Address auto suggestion mode for this subnet.')
```

Comment:
Should we still list all the valid modes? e.g something like:
```suggestion
          param :ipam, IPAM::MODES.values, :desc => (N_("IP Address auto suggestion mode for this subnet. Valid values are: %s") %  IPAM::MODES.values.join(','))
```
Or is this something apipie does already when using a list validator?
```

Reply:
That's how it looks after my patch:

![Screenshot from 2019-11-11 11-33-19](https://user-images.githubusercontent.com/94284/68580658-1a795500-0477-11ea-88d1-6894efafe7f5.png)

Reply:
Also, making the description dynamic makes it hard for translations :)